docker/production/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY requirements /requirements
+RUN pip install -r /requirements/dist.txt
```

Comment:
If I understand correctly, dependencies are also placed under /app/requirements. If so, why do you copy them separately?